src/DayColumn.js

```diff
@@ -48,9 +48,12 @@ class DayColumn extends React.Component {
       this.clearTimeIndicatorInterval()
 
       if (this.props.isNow) {
-        this.setTimeIndicatorPositionUpdateInterval(
+        const tail =
+          this.props.getNow === this.props.getNow &&
```

Comment:
this shouldn't compare the function but the result of the function. e.g the actual `now` value. Also you are comparing getNow with it self here...

I don't actually think this is needed, it should already update if now changes

Reply:
It won't work in a scenario when `isNow` doesn't change, but `getNow` does, because few lines earlier there's a check if `isNow` has changed:
https://github.com/intljusticemission/react-big-calendar/blob/1c101aae13f4a16426685208ab7403ba05c4c378/src/DayColumn.js#L47

> Also you are comparing getNow with it self here...

My bad, meant to compare it with previous value :wink: 
Check out the second commit, I've restructured the code a bit.

Reply:
@jquense any updates on this one?